deployment/lightgbm/lightgbm-house-price-prediction.ipynb

```diff
@@ -0,0 +1,566 @@
+        "        }\n",
+        "        \n",
+        "    def example(self):\n",
+        "        return [[3.0, 1.0, 206.0, 110.0, 8.4, 1980.0, 1.0, 8801.0]]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "6T8uAo2fQy9J"
+      },
+      "outputs": [],
+      "source": [
+        "artifacts = {'serialized_model': 'model.pkl'}"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "-e9vkQILQ1H2",
+        "outputId": "d8ebc1de-9ede-4ed4-99e8-4821658eb2ba"
+      },
+      "outputs": [],
+      "source": [
+        "registered_model = client.get_or_create_registered_model(name='House Price Prediction')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "85cbJc3VRAkU",
+        "outputId": "30801f82-35ba-4f3a-9ef6-788dbf248c05"
+      },
+      "outputs": [],
+      "source": [
+        "model = registered_model.create_standard_model(\n",
+        "    name = 'lightgbm',\n",
+        "    model_cls = Predictor,\n",
+        "    environment = Python(requirements=['scikit-learn', 'lightgbm']),\n",
+        "    model_api = ModelAPI(X_train, y_train.to_frame(name='PricePrediction')),\n",
+        "    artifacts = artifacts\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "f4jpOPlqQouG"
+      },
+      "source": [
+        "## 6. Model Endpoint"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "hixy1I8uRGI6"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint = client.get_or_create_endpoint('house-price-prediction-lightgbm')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "HPWWwP1jRJ4M",
+        "outputId": "270c50f0-b3f0-455b-af8d-7093db1ee57f"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint.update(model, wait=True)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "7nvkzyzTRMVX"
+      },
+      "outputs": [],
+      "source": [
+        "deployed_model = endpoint.get_deployed_model()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tISSQrzFWbA2"
+      },
+      "source": [
+        "## 7. Predictions"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "N4GOi5ucWywL"
+      },
+      "outputs": [],
+      "source": [
+        "sample_data = X_train.values.tolist()[:5]"
```

Comment:
I think we should use `X_test` here, because it is available in the notebook and the model is already trained (and then maybe we also don't need `cols` below anymore)

Reply:
Changed from `X_train` to `X_test` :+1:

I still need to specify the columns, as `sample_data = X_test.values.tolist()[:5]` returns a list containing only the DataFrame values (excluding its columns names).

Even so, the code got cleaner with: `df = pd.DataFrame(data=sample_data, columns=X_test.columns)`
The cell that was creating a list with columns names is not necessary.